src/main/java/duke/command/Parser.java

```diff
@@ -0,0 +1,59 @@
+package duke.command;
+
+import duke.exception.DukeException;
+import duke.task.TaskType;
+
+public class Parser {
+    private static final String COMMAND_BYE = "bye";
+
+    private static final String COMMAND_LIST = "list";
+
+    private static final String COMMAND_MARK = "mark";
+    private static final String COMMAND_UNMARK = "unmark";
+
+    private static final String COMMAND_TODO = "todo";
+    private static final String COMMAND_DEADLINE = "deadline";
+    private static final String COMMAND_EVENT = "event";
+
+    private static final String COMMAND_DELETE = "delete";
+
+    private static final String COMMAND_FIND = "find";
+
+    private static final String ERROR_INVALID_COMMAND = "OOPS!!! You have entered an invalid command :(";
+
+    /**
+     * Parser to process the entered comments.
```

Comment:
Should the javadocs be in the form of 'Parses...'?

Reply:
thanks for the suggestion! will fix this